.github/workflows/build.yml

```diff
@@ -15,11 +15,17 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
-      - name: Install dependencies
+      - name: Install Dependencies
         run: |
           python -m pip install --upgrade pip
+          pip install pip-autoremove
           pip install flit
-      - name: Test build
+      - name: Test Package Installation
         run: |
           make install
+      - name: Remove Aux. Dependencies
```

Comment:
whats going on with this remove? also the naming is a bit confusing test package installation isnt really a test its just installation. test cli endpoints is running test-installation. Maybe change the name of the earlier step to install package and then the last step to test installation/cli endpoints or something along those lines